### ISSUE TITLE
Wiki Documents

### ISSUE THREAD

**Issue by snotlingorc** [TheNAF/naflm#288]
I was thinking about the wiki pages some. What would be nice are some guides (play books?) for each the Site Admin, Commissioner, Coach and Developer.  Separating out each one with a directory structure like :
![directory](https://user-images.githubusercontent.com/11547344/88463982-37727e80-ce74-11ea-8f97-626946c2b5e2.PNG)

The main page could look something like:
![main](https://user-images.githubusercontent.com/11547344/88463969-13af3880-ce74-11ea-87f2-ebcb4ffa71d2.PNG)

Wondering what thoughts anyone has about the wiki and how it could be used/setup.

**Comment by snotlingorc** [comment]
Well.. you kind of can use directories, you just do not see them.  I put up an example here : https://github.com/snotlingorc/docker-naflm/wiki.

**Comment by snotlingorc** [comment]
I did not see the amazing pdf in the downloads section.  That was basically everything I was thinking about putting in the wiki. Awesome job!  I went and took the pdf and moved it into the wiki/markdown format.  https://github.com/snotlingorc/docker-naflm/wiki as an example.  

If you want to push this into the naflm wiki, Clone it from https://github.com/snotlingorc/docker-naflm.wiki.git and push it to https://github.com/TheNAF/naflm.wiki.git

**Comment by thefloppy1** [comment]
Yeah the PDF is handy, we need to make sure that people see it more, I need
to add it to the wiki, however I am flat out with home schooling, can
somebody please do this for us.
